l1-contracts/src/core/interfaces/IRollup.sol

```diff
@@ -91,6 +93,8 @@ struct RollupConfig {
   IInbox inbox;
   IOutbox outbox;
   uint256 version;
+  uint256 entryQueueFlushSizeMin;
```

Comment:
Point of interest for gas savings here. Config generally can be reduced a good chunk 😬 But think that is things separate to just these as well, so created #15032